### DISCUSSION TITLE
Ender 3 v2 Laser Engraver

### DISCUSSION THREAD

**Discussion by LexusSpeed03** [Jyers/Marlin#1076] [Q&A] [answered]
Hey Guys,

I haven't seen much discussion but I have the laser engraver installed with a bracket to allow the BL Touch and it installed at the same time. However I have followed the steps of the guide to setup the laser engraver but when it goes to print it tries to move the bed right off the start and sounds like the belt jumping.

This being said will the Jyers firmware work I am currently using newest version with BL Touch HS 5x5.

Any thoughts would be greatly appreciated.

**Comment by smooreace** [comment]
Do you have the proper probe offsets... set?

**Comment by Jyers** [comment, marked as answer]
Make sure you set the home position using the home menu. For a laser system I believe you usually set it based on your work, so you need to move the head where you need it and select set home position.

**Reply by LexusSpeed03** [reply]
Alright let me try that I would really like to get it working so I can use this for dual purpose and not keep switching around.
